includes/class-wc-siftscience-admin.php

```diff
@@ -499,6 +502,13 @@ private function create_element( $type, $id, $label = '', $desc = '', $element_o
 
 			$custom_attributes = array(); // array flattener.
 
+			if ( WC_SiftScience_Html::WC_SELECT_ELEMENT === $type ) {
```

Comment:
This made me laugh :-)

Reply:
this is validation not creation **you requested this you should laugh**